docs/changelog-fragments.d/README.md

```diff
@@ -57,23 +50,20 @@ Added a `{user}` role to Sphinx config -- by {user}`webknjaz`
 File `docs/changelog-fragments.d/116.feature.md`:
 
 ```md
-Added support for nested module options (suboptions)
--- by {user}`tomaciazek`
```

Comment:
This change doesn't make sense, it was demonstrating that it is okay to have more than one line in fragments.

Reply:
That is clearly an unbacked statement as there is no need need to wrap text to show people they can wrap it or not. We are not in the business to teach people markdown syntax.

If you really wanted to demonstrate it you would have included something like `This can also wrap on multiple lines." as part of the fragment itself.

As we would introduce automatic formatting there we have no need to tell user how and where to wrap the text as that will be done automatically.